src/test/java/lotto/InsightResultTest.java

```diff
@@ -0,0 +1,38 @@
+package lotto;
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package lotto.domain;
```